protobuf-maven-plugin/src/it/gh-565-attach-descriptor-file/attach-custom-classifier/pom.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
In src/test/java there is an AbstractGenerateMojo test template that has a load of test cases in it that just verify the Maven params get added to the generation request correctly. Would it be possible to add some test cases there? 

(Appreciate that class is a mess, it is on my list of things to eventually find time to improve).

Reply:
added test cases